lib/script/script.js

```diff
@@ -317,6 +318,21 @@ Script.prototype.isPublicKeyHashOut = function() {
     this.chunks[4].opcodenum === Opcode.OP_CHECKSIG);
 };
 
+/**
+ * @returns {boolean} if this is a pay to pubkey hash 256 output script
+ */
+Script.prototype.isPublicKeyHash256Out = function() {
+  var isha256Index = this.chunks.length === 5 ? 0 : 8;
```

Comment:
Hmm, I would split the coldstaking out and the publickeyhash256out.
On the first pass of the code, I didn't realize that the constants (5 and 14) are used to differentiate between a normal PublicKeyHash256 and a coldstaking output.

Reply:
@spazzymoto did you read this?